app/main/controllers/projects.py

```diff
@@ -158,54 +183,105 @@ def get_project_sources(user_id, project_id):
             })
 
         pattern = '%' + search_query + '%'
-        results = Source.query.filter(Source.project_id == project_id)\
+        results = Source.query.filter(Source.project_id == project_id) \
             .filter(Source.url.ilike(pattern)
                     | Source.title.ilike(pattern)
                     | Source.content.ilike(pattern)
-                    | Source.highlights.ilike(pattern)
-                    | Source.notes.ilike(pattern)).order_by(Source.id).all()
+                    | Source.child_items.ilike(pattern)).order_by(Source.id).all()
 
         return jsonify({
             'success': True,
             'sources': [source.format_short() for source in results]
         })
 
+
+
+    """
+    Gets all the items of a given project
+    or searches through them if a search query is passed.
+    Search looks at all text fields of a source.
     """
-    Creates a new source inside an existing project.
+    @app.route('/projects/<int:project_id>/items')
+    @requires_auth('read:items')
+    def get_project_items(user_id, project_id):
+        project = get_authorized_project(user_id, project_id)
+
+        search_query = request.args.get('query', None)
+        if search_query is None:
+            # Returns all items
+            return jsonify({
+                'success': True,
+                'sources': [i.format() for i in project.items]
```

Comment:
Probably don't want to return items by the `sources` keyword, it'd be better to also call them items.
```suggestion
                'items': [i.format() for i in project.items]
```